src/server/supplements/har/harTracer.ts

```diff
@@ -20,6 +20,9 @@ import { helper } from '../../helper';
 import * as network from '../../network';
 import { Page } from '../../page';
 import * as har from './har';
+import * as types from '../../types';
+
+const FALLBACK_HTTP_VERSION = 'http/1.1';
```

Comment:
I believe HTTP (capital case) is more common and that is also what it was before, let's keep it?